src/fides/api/alembic/migrations/versions/fd52d5f08c17_system_dictionary_data_migration.py

```diff
@@ -0,0 +1,374 @@
+            DPA_LOCATION: dpa_location,  # From System.data_protection_impact_assessment
+            DPA_PROGRESS: dpa_progress,  # From System.data_protection_impact_assessment
+        }
+
+        update_system_query: TextClause = text(
+            """
+                UPDATE ctl_systems
+                SET 
+                    joint_controller_info = :joint_controller_info,
+                    does_international_transfers = :does_international_transfers,
+                    responsibility = :responsibility,
+                    requires_data_protection_assessments = :requires_data_protection_assessments,
+                    dpa_location = :dpa_location,
+                    dpa_progress = :dpa_progress
+                WHERE id = :id;
+            """
+        )
+
+        system_data: Dict[str, Any] = system_data
+        bind.execute(
+            update_system_query,
+            {"id": row["id"], **system_data},
+        )
+
+
+# Casing is changing
+legal_basis_for_processing_map: Dict = {
+    "Consent": "Consent",
+    "Contract": "Contract",
+    "Legal Obligation": "Legal obligations",
+    "Legal obligations": "Legal obligations",
+    "Vital Interest": "Vital interests",
+    "Vital interests": "Vital interests",
+    "Public Interest": "Public interest",
+    "Legitimate Interests": "Legitimate interests",
+    "Legitimate interests": "Legitimate interests",
+}
+
+# Language/casing is changing
+special_category_map: Dict = {
+    "Consent": "Explicit consent",
+    "Employment": "Employment, social security and social protection",
+    "Vital Interests": "Vital interests",
+    "Non-profit Bodies": "Not-for-profit bodies",
+    "Public by Data Subject": "Made public by the data subject",
+    "Legal Claims": "Legal claims or judicial acts",
+    "Substantial Public Interest": "Reasons of substantial public interest (with a basis in law)",
+    "Medical": "Health or social care (with a basis in law)",
+    "Public Health Interest": "Public health (with a basis in law)",
+}
+
+
+def _get_privacy_declaration_legal_basis_for_processing(
+    data_use_legal_basis: Optional[str],
+    legitimate_interest_impact_assessment: Optional[str],
+) -> Optional[str]:
+    """Use DataUse.legal_basis or DataUse.legitimate_interest_impact_assessment to set the value of
+    PrivacyDeclaration.legal_basis_for_processing"""
+    if legitimate_interest_impact_assessment:
+        return "Legitimate interests"
+    if data_use_legal_basis:
+        return legal_basis_for_processing_map.get(data_use_legal_basis, None)
+    return None
+
+
+def _get_third_party_data(
+    recipients: Optional[List[str]],
+) -> Tuple[bool, Optional[str]]:
+    """Migrate data from DataUse.recipients field to new PrivacyDeclaration.third_parties and
+    PrivacyDeclaration.data_shared_with_third_parties be true
+    """
+    data_shared_with_third_parties: bool = False
```

Comment:
```suggestion
    data_shared_with_third_parties: bool = recipients and isinstance(recipients, list)
```

Reply:
so how about bool(recipients) here and I'll leave the list check below